### ISSUE TITLE
Rebrand @Mozteach to @MozillaLearning

### ISSUE THREAD

**Issue by KGorr** [mozilla/learning-networks#405]
To align with broader branding strategy, we need a plan for rebranding MozTeach twitter handle to MozillaLearning.

5 step communication plan [outlined here](https://docs.google.com/document/d/1ja0wZm9XvMo2daRg7612w-14Meg1jNZFYkJ5i8m_XYc/edit#).

CC @ldecoursy, @KevZawacki, @HPaulJohnson

**Comment by KevZawacki** [comment]
@KGorr Looks great -- comprehensive! Left a couple small comments.

**Comment by ldecoursy** [comment]
cc @mcdermott415 @chrislarry33

**Comment by KGorr** [comment]
Thanks everyone for the input! We're going to start moving on this so that we can communicate the change during next week's community call. 

Today, I'm going to complete Step 1(transition prep), which means @hannahkane, I'll file a separate ticket for the @mozteach references that need to be made on the site after Step 2 is complete. 

Step 2 (transition) will be completed on Monday, May 23.

**Comment by KGorr** [comment]
The transition (step 2) has been completed. The rest is just reminding folks and monitoring for questions, so I'm closing this ticket. Thank you!
